src/common/snippets/src/utils/utils.cpp

```diff
@@ -93,33 +121,34 @@ auto get_non_scalar_constant_count_for_fq(const std::shared_ptr<ov::op::v0::Fake
     // Some of them can be scalar or non-scalar. It depends on which original 4 Constants are non-scalar
     // To sum it up, below conditions check all possible cases to calculate count of new generated non-scalars
     if (is_optimized) {
-        if (il && ih)
+        if (il && ih) {
             return 3;
-        else if (il || ih)
-            return 2;
-        return 0;
-    } else {
-        if (ol && il && ih)
-            return 6;
-        else if ((ol && (il || ih)) || (il && ih && oh))
-            return 5;
-        else if ((il && oh) || (ih && oh) || (il && ih))
-            return 4;
-        else if (il || ih)
-            return 3;
-        else if (ol)
+        }
+        if (il || ih)
             return 2;
-        else if (oh)
-            return 1;
         return 0;
     }
+    if (ol && il && ih)
+        return 6;
+    else if ((ol && (il || ih)) || (il && ih && oh))
+        return 5;
+    else if ((il && oh) || (ih && oh) || (il && ih))
+        return 4;
+    else if (il || ih)
+        return 3;
+    else if (ol)
+        return 2;
+    else if (oh)
+        return 1;
+    return 0;
```

Comment:
```suggestion
    if ((ol && (il || ih)) || (il && ih && oh))
        return 5;
    if ((il && oh) || (ih && oh) || (il && ih))
        return 4;
    if (il || ih)
        return 3;
    if (ol)
        return 2;
    if (oh)
        return 1;
    return 0;
```
And it doesn't clear why check on braces didn't work here